portfolio/src/main/java/com/google/sps/servlets/DataServletDelete.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that allows users to add comments and returns comment history  */
+@WebServlet("/delete-comments")
```

Comment:
It's not introduced in the walkthroughs AFAIK, but there's a better HTTP method for this: DELETE. Since we're still operating on comments here, we can move the handler to the same class as the other comment handlers and reuse the /comments URL with the DELETE method to perform this function.

Reply:
Great, I'll look into this! I'll try to incorporate the single comment delete feature into this as well.

Reply:
Sounds good; looks like you've done that in #17.